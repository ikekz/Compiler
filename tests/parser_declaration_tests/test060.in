type
	t = array [1..10] of array [-6..0] of double;
begin
end.