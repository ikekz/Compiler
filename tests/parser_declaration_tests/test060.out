integer        type           integer
double         type           double
char           type           char
t              type           array [1..10] of array [-6..0] of double
